### ISSUE TITLE
"color" is ambiguous and needs a definition (oj04fd)

### ISSUE THREAD

**Issue by daniel-montalvo** [act-rules/act-rules.github.io#1603]
[Rule](https://act-rules.github.io/rules/oj04fd)

[Related CG discussion](https://www.w3.org/2020/11/26-act-r-minutes.html#item04)

Since WCAG does not define color, we need to find a workaround for this. Lightness might be also considered. See technique [F73: Failure of Success Criterion 1.4.1 due to creating links that are not visually evident without color vision(https://www.w3.org/WAI/WCAG21/Techniques/failures/F7)

**Comment by WilcoFiers** [comment]
Two actions needed:
1. Change the use of color in this rule to "relative luminance"
2. Look at other rules that use the word color, and figure out what to substitute it with

**Comment by ajanec01** [comment]
Ahh, I don't think this will be great either. Let's have a look at the two values:
- `hsl(0, 100%, 50%)`; and 
- `hsl(210, 100%, 50%)`.
The first value is red and the second is blue as on the image below. The difference in relative luminance is 1:1, which means that this scenario would not be caught by the rule. 
![image-of-re-and-blue](https://user-images.githubusercontent.com/47136246/118172876-cc0a9500-b424-11eb-99d6-f2745d0b423f.png)

I suggest:

> For each target element, there is at least one device pixel inside the scrolling area of the viewport whose hue value or relative luminance is different when the element is focused from when it is not.
